app/src/main/java/org/fossasia/susi/ai/dataclasses/SkillsBasedOnMetrics.kt

```diff
@@ -8,5 +8,6 @@ import org.fossasia.susi.ai.rest.responses.susi.SkillData
  */
 data class SkillsBasedOnMetrics(
         var metricsList: ArrayList<List<SkillData>?>,
-        var metricsGroupTitles: ArrayList<String>
+        var metricsGroupTitles: ArrayList<String>,
+        var groups: MutableList<String>
```

Comment:
Can't these be immutable?

Reply:
If I use ArrayList<String> then it can be.

Reply:
But, the response coming from the server has type List<String> as you had asked me to use List instead of String. Thus, I used MutableList<String>. Would it be fine if I use ArrayList to for the list received from the server?